src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -841,14 +849,16 @@ private PackageLicenseInfo GeneratePackageLicenseInfo(IPackageSearchMetadata met
             UserAction userAction,
             bool removeDependencies,
             bool forceRemove,
-            CancellationToken token)
+            CancellationToken token,
+            string? newMappingID = null,
+            string? newMappingSource = null)
```

Comment:
A `CancellationToken` parameter, if present, is typically last (like [here](https://github.com/NuGet/NuGet.Client/pull/5109/files#diff-77c5ac57c969eaa5ab522f95d16bbf96ab31f78a256489bd165a5b72c3500aeaR444-R446).

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1068